experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -4,67 +4,52 @@
 
 import 'package:flutter/material.dart';
 
-class ComponentScreen extends StatelessWidget {
-  const ComponentScreen({super.key, required this.showNavBottomBar});
+class FirstComponentList extends StatelessWidget {
+  const FirstComponentList({
+    super.key,
+    required this.showNavBottomBar,
+    required this.scaffoldKey,
+    required this.showSecondList,
+  });
 
   final bool showNavBottomBar;
+  final GlobalKey<ScaffoldState> scaffoldKey;
+  final bool showSecondList;
 
   @override
   Widget build(BuildContext context) {
-    return Expanded(
-      child: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 10),
-        child: Align(
-          alignment: Alignment.topCenter,
-          child: SizedBox(
-            width: maxWidthConstraint,
-            child: ListView(
-              shrinkWrap: true,
-              children: [
-                colDivider,
-                colDivider,
-                const Buttons(),
-                colDivider,
-                colDivider,
-                const IconToggleButtons(),
-                colDivider,
-                const FloatingActionButtons(),
-                colDivider,
-                const Chips(),
-                colDivider,
-                const Cards(),
-                colDivider,
-                const TextFields(),
-                colDivider,
-                const Dialogs(),
-                colDivider,
-                const Switches(),
-                colDivider,
-                const Checkboxes(),
-                colDivider,
-                const Radios(),
-                colDivider,
-                const ProgressIndicators(),
-                colDivider,
-                showNavBottomBar
-                    ? const NavigationBars(
-                        selectedIndex: 0,
-                        isExampleBar: true,
-                      )
-                    : Container(),
-              ],
-            ),
-          ),
-        ),
-      ),
+    return ListView(
+      shrinkWrap: true,
```

Comment:
This is probably fine for now, but we should really look into finding a way around using `shrinkWrap` here as it can lead to performance issues and we don't really want to encourage its use in an example. You might talk this over with @Piinks and see if she has any suggestions here.

Reply:
Actually, I'd love to take a look at this first. We are trying to not recommend shrinkwrap unless absolutely necessary and have been removing it from sample code. It is very costly to performance and should only be used if there isn't an option to use slivers. I'll be able to take a look today and see if we can remove this.